public/js/btn_back_to_top.js

```diff
@@ -0,0 +1,14 @@
+window.onscroll = function() {scrollFunction()};
```

Comment:
Outra coisa, cuidem com essas atribuições diretas de `eventListeners`, pois vocês podem fazer overriding de eventos que foram previamente atribuídos em outros momentos, causando bugs. Caso for preciso, usem `addEventListener`, que permite adicionar múltiplos listeners para cada evento de um elemento.